### PR TITLE
test: check active channel is correct

### PR DESCRIPTION
**Pull request by seantking**
## Description

Small PR to add check that active channel is set correctly. 

closes: #314 

---

Before we can merge this PR, please make sure that all the following items have been
checked off. If any of the checklist items are not applicable, please leave them but
write a little note why.

- [x] Targeted PR against correct branch (see [CONTRIBUTING.md](https://github.com/cosmos/ibc-go/blob/master/CONTRIBUTING.md#pr-targeting))
- [x] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [x] Code follows the [module structure standards](https://github.com/cosmos/cosmos-sdk/blob/master/docs/building-modules/structure.md).
- [x] Wrote unit and integration [tests](https://github.com/cosmos/ibc-go/blob/master/CONTRIBUTING.md#testing)
- [ ] Updated relevant documentation (`docs/`) or specification (`x/<module>/spec/`)
- [ ] Added relevant `godoc` [comments](https://blog.golang.org/godoc-documenting-go-code).
- [ ] Added a relevant changelog entry to the `Unreleased` section in `CHANGELOG.md`
- [x] Re-reviewed `Files changed` in the Github PR explorer
- [x] Review `Codecov Report` in the comment section below once CI passes

### CODE REVIEW

File: modules/apps/27-interchain-accounts/keeper/handshake_test.go [hunk 1]
```diff
@@ -221,12 +221,15 @@ func (suite *KeeperTestSuite) TestOnChanOpenAck() {
 				path.EndpointA.ChannelConfig.PortID, path.EndpointA.ChannelID, counterpartyVersion,
 			)
 
+			expectedActiveChannel := "channel-0"
```
Comment (colin-axner) [inline modules/apps/27-interchain-accounts/keeper/handshake_test.go:224]: could we use the `path.EndpointA.ChannelID` as the expected active channel?

File: modules/apps/27-interchain-accounts/keeper/handshake_test.go [hunk 2]
```diff
@@ -214,19 +220,23 @@ func (suite *KeeperTestSuite) TestOnChanOpenAck() {
...
 
 			tc.malleate() // explicitly change fields in channel and testChannel
 
 			err = suite.chainA.GetSimApp().ICAKeeper.OnChanOpenAck(suite.chainA.GetContext(),
 				path.EndpointA.ChannelConfig.PortID, path.EndpointA.ChannelID, counterpartyVersion,
 			)
 
+			activeChannel, _ := suite.chainA.GetSimApp().ICAKeeper.GetActiveChannel(suite.chainA.GetContext(), path.EndpointA.ChannelConfig.PortID)
+
+			suite.Require().Equal(activeChannel, expectedChannel)
```
Comment (colin-axner) [inline modules/apps/27-interchain-accounts/keeper/handshake_test.go:233]: maybe in tc.expPass this should be expected Channel and in the else it should be an empty string?

**Review by colin-axner — APPROVED** [review]
Nice work! Just left one suggestion

File: modules/apps/27-interchain-accounts/keeper/handshake_test.go (hunk 2 above)
Comment (seantking) [reply modules/apps/27-interchain-accounts/keeper/handshake_test.go:233]: Ah sorry, this was on auto-merge :D

Comment (seantking) [reply modules/apps/27-interchain-accounts/keeper/handshake_test.go:233]: Yeah, your suggestion prob reduces the need to copy pasta `expectedChannel = ""` in the malleate functions. I'll add it into another PR.

**Pull request merged**